Source/Dafny/DafnyAst.cs

```diff
@@ -7973,6 +7986,19 @@ public GuardedAlternative(IToken tok, bool isBindingGuard, Expression guard, Lis
       this.IsBindingGuard = isBindingGuard;
       this.Guard = guard;
       this.Body = body;
+      this.Attributes = null;
+    }
+    public GuardedAlternative(IToken tok, bool isBindingGuard, Expression guard, List<Statement> body, Attributes attrs)
```

Comment:
Are there any callers of the previous `GuardedAlternative` constructor? If not, keep just this new one.
(The same comment applies to the other new constructors in this file.)